frontend/isoko/src/components/BusinessPreview.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Rating } from 'react-simple-star-rating';
+import { Table } from 'react-bootstrap';
+import MinorityTag from './MinorityTag';
+import KeywordTag from './KeywordTag';
+
+const BusinessContainer = styled(Table)`
+   width: 85%;
```

Comment:
When styling components it is important to remember that the component might not be used in the same place always. So although the width 85% looks alright on the home page, it would be better to change this to 100% and then change the margin-left and margin-right to be 10% in the FeaturedBusinesses styled component on the home page.